CONTRIBUTING

```diff
@@ -1,191 +1,5 @@
-Licensing
-=========
+We welcome any type of contribution. Have a look at our additional information, 
+and please open an issue before starting any major activity, so that we can discuss 
```

Comment:
I think that the link within the text would be more visible. Otherwise you can add a reference to make it more explicit, e.g.:
```
additional information [1] ....

[1] http:///
```